### PR TITLE
Corrected the typos, severity levels and removed info alerts

### PR DESCRIPTION
**Pull request by shtripat**
ceph-mixins-issue: #34 
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@anmolsachan @umangachapagain please review and approve. These are changes as per UX guidelines.

**Review by julienlim — COMMENTED** [review]
LGTM.

File: extras/manifests/prometheus-rules.yaml [hunk 1]
```diff
@@ -155,7 +108,7 @@ spec:
         severity: warning
     - alert: CephClusterCriticallyFull
       annotations:
-        description: The utilization of storage cluster has crossed 95%.
+        description: Storage cluster utilization has crossed 95%.
         message: Storage cluster is critically full and needs immediate expansion
```
Comment (anmolsachan) [inline extras/manifests/prometheus-rules.yaml]: Full stop in the end :P

File: extras/manifests/prometheus-rules.yaml [hunk 2]
```diff
@@ -132,7 +85,7 @@ spec:
     - alert: CephClusterWarningState
       annotations:
         description: Storage cluster is in warning state for more than 10m.
-        message: Storage cluster is in warning state
+        message: Storage cluster is in degraded state
```
Comment (anmolsachan) [inline extras/manifests/prometheus-rules.yaml]: Full stop in the end

**Comment by anmolsachan** [comment]
PR looks good to me . Just some full stops missing in message and description field. Not sure if they are intended, but should be uniform

File: alerts/utilization.libsonnet [hunk 3]
```diff
@@ -14,8 +14,8 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Storage cluster is nearing full. An expansion is required',
-              description: 'The utilization of storage cluster has crossed 85%.',
+              message: 'Storage cluster is nearing full. Expansion is required.',
```
Comment (anmolsachan) [inline alerts/utilization.libsonnet:17]: May be we can write "Storage cluster expansion is required"

File: alerts/utilization.libsonnet [hunk 4]
```diff
@@ -14,8 +14,8 @@
...
+              description: 'Storage cluster utilization has crossed 85%.',
```
Comment (anmolsachan) [inline alerts/utilization.libsonnet:18]: we can add "Storage cluster expansion is required". Same for 95 % alert as well

File: alerts/utilization.libsonnet [hunk 5]
```diff
@@ -31,7 +31,7 @@
             },
             annotations: {
               message: 'Storage cluster is critically full and needs immediate expansion',
-              description: 'The utilization of storage cluster has crossed 95%.',
+              description: 'Storage cluster utilization has crossed 95%.',
```
Comment (anmolsachan) [inline alerts/utilization.libsonnet:34]: we can add "Storage cluster expansion is required immediately.

File: alerts/utilization.libsonnet (hunk 3 above)
Comment (shtripat) [reply alerts/utilization.libsonnet:17]: I feel `Storage cluster` twice in same message doesn't look good. I will still go with existing message.

File: alerts/utilization.libsonnet (hunk 4 above)
Comment (shtripat) [reply alerts/utilization.libsonnet:18]: Same as above

File: alerts/utilization.libsonnet (hunk 5 above)
Comment (shtripat) [reply alerts/utilization.libsonnet:34]: Same as above

**Comment by shtripat** [comment]
@anmolsachan `message` field we are not ending with `.` Only `description` field we will end with `.`

**Comment by julienlim** [comment]
+1 on message not ending with . but description should end with .

@shtripat Can you pls address? Thanks.

**Comment by shtripat** [comment]
> +1 on message not ending with . but description should end with .
> 
> @shtripat Can you pls address? Thanks.

@julienlim all the description feilds are ending with `.`

**Review by umangachapagain — APPROVED** [review]

**Review by anmolsachan — APPROVED** [review]

**Pull request merged**